test/tapes/response-239.json5

```diff
@@ -0,0 +1,69 @@
+{
```

Comment:
Looks like related image requests don't have custom tape names :cry:

Reply:
This is actually a request for an image with an id that doesn't exist (`foo`). Since the tape name are generated using a UUID regex, this is expected. We could edit the regex to be more broad so that this specific case is caught, too.

Reply:
Ahhhh, okay! Makes sense then :slightly_smiling_face: